doc/api/promise_hooks.md

```diff
@@ -0,0 +1,181 @@
+# Promise hooks
+
+<!--introduced_in=REPLACEME-->
+
+> Stability: 1 - Experimental
+
+<!-- source_link=lib/async_hooks.js -->
+
+The `promiseHooks` API is part of the `async_hooks` module:
```

Comment:
Maybe add some words when to use promise hooks and when async_hooks and what the difference is.

In special we should avoid that people think that e.g. promise hooks are helpful in addition to an async_hook or for async local storage usecases.